src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/SampleData.kt

```diff
@@ -298,6 +298,13 @@ class SampleData {
       return AuthUser(id, authSource, userName)
     }
 
+    fun sampleCancellationReason(
```

Comment:
let's add this to the factory classses instead since we decided to try to use those methods more consistently